packages/office-ui-fabric-react/src/utilities/hooks/useResponsiveMode.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { getWindow } from '@uifabric/utilities';
+import { useOnEvent } from '@uifabric/react-hooks';
+import { ResponsiveMode, getResponsiveMode } from '../decorators/withResponsiveMode';
+
+/**
+ * User specified mode to default to, useful for server side rendering scenarios.
+ */
+let _defaultMode: ResponsiveMode | undefined;
```

Comment:
This isn't doing anything right now. Ideally you should find a way to share this (and maybe also _lastMode) between the decorator and the hook.